bin/smart-proxy-win-service

```diff
@@ -2,41 +2,42 @@ $LOAD_PATH.unshift(*Dir[File.expand_path("../../lib", __FILE__), File.expand_pat
 
 raise "win-service executable can only be used as a service in windows environment" unless RUBY_PLATFORM =~ /mingw/
 
+require 'rubygems'
+require 'bundler/setup'
 require 'win32/daemon'
-require 'logger'
 require 'smart_proxy_main'
 
 include Win32
 
 LOG_DIR_PATH = File.expand_path("../../log", __FILE__)
 Dir.mkdir(LOG_DIR_PATH) unless Dir.exist?(LOG_DIR_PATH)
 
-def logger
-  @logger ||= Logger.new(File.expand_path("win-service.log", LOG_DIR_PATH))
```

Comment:
Opening a file directly is more reliable (it's amazing how much grief these few lines of code give me).